BranchMetrics/android-branch-deep-linking-attribution
# CORE-1355/remove close api

## Reference
[CORE-1355](https://branch.atlassian.net/browse/CORE-1355
) Remove /v1/close call from Android SDK

## Description
Stop calling /v1/close from the Android SDK whenever a session ends. We don't make use of the call anywhere in our backend and it is costing us money in CDN and gateway expenses.

## Testing Instructions
just make sure that you can still open and close the app in various ways and the app/install requests are still fired. Also make sure that close request is NOT being fired upon backgrounding the app.

## Risk Assessment [`HIGH` || `MEDIUM` || `LOW`]
LOW


## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style
